scripts/release/operatorhub/test.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -euo pipefail
+
+show_help() {
+  echo "test - runs Operator Hub tests"
+  echo " "
+  echo "./test.sh [options]"
+  echo " "
+  echo "Options:"
+  echo "-h, --help              shows brief help"
+  echo "-t, --test              runs operator-framework ansible tests (default: all. other supported values are: orange, kiwi or lemon)"
+}
+
+tests=all
+
+while test $# -gt 0; do
+  case "$1" in
+    -h|--help)
+        show_help
+        exit 0
+        ;;
+    -t)
+        shift
+        if test $# -gt 0; then
+          tests=$1
+        fi
+        shift
+        ;;
+    --test*)
+        tests=$(echo $1 | sed -e 's/^[^=]*=//g')
+        if [[ "$tests" == "--test" ]]; then
+          tests=all
+        fi
+        shift
+        ;;
+    *)
+        echo "Unknown param $1"
+        exit 1
+        break
+        ;;
+  esac
+done
+
+CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+BUNDLE_DIR="${CUR_DIR}"/../../../bundle
+
+GITHUB_TOKEN="${GITHUB_TOKEN:-}"
```

Comment:
These are all the same variables arn't they? Source them?

Reply:
As in have a shared file between `publish` scripts? Not sure we gain much but we make it a bit more difficult to navigate.

Reply:
maybe.. ok